mkosi/resources/mkosi.md

```diff
@@ -790,6 +790,18 @@ they should be specified with a boolean argument: either "1", "yes", or "true" t
   (the one with the highest version) and stored in
   `EFI/BOOT/BOOTX64.EFI`.
 
+`BiosBootloader=`, `--bios-bootloader`
+
+: Takes one of `none`, `auto` and `grub`. Defaults to `auto`. If set to
+  `none`, no BIOS bootloader will be installed. If set to `auto`, if
+  grub is installed and a BIOS boot partition is configured, grub will
```

Comment:
This should mention `21686148-6449-6e6f-744e-656564454649` (and maybe also mention that 1MB is enough) so that people know what this means, maybe we should also call this a _GRUB partition_ instead of a BIOS boot partition, so that it's really clear to users, that this is a GRUB thing and doesn't have anything to do with `/boot`.